components/builder-api-client/src/lib.rs

```diff
@@ -47,9 +47,9 @@ use std::string::ToString;
 
 use broadcast::BroadcastWriter;
 use chrono::DateTime;
-use hab_core::package::{PackageIdent, Identifiable, PackageArchive};
-use hab_http::ApiClient;
+use hab_core::package::{Identifiable, PackageArchive, PackageIdent};
```

Comment:
Wait, is `Identifiable` still being used?

Reply:
It shouldn't be but it looks like we missed something here. Will get this corrected as well.

Reply:
I think this is _probably_ ok. I haven't heard from fletcher on it yet. But I'm guessing since we _are_ in fact using the appropriate type now and just leveraging the trait to get access to some of the PackageIdent methods it's less of an issue.

Reply:
Yep, that why we still need to `use …::Identifiable`: without it we don't get the method accessors like `.origin()` and the `.fully_qualified()` sanity checker method. I'm hopeful that with a bit of refactoring, those methods can be tacked on `PackgeIdent` directly and thus we'd eventually drop the `use` of that trait here.